issue templates im github repos hinterlegen (#2107)
redaxo/redaxo: .github/ISSUE_TEMPLATE/---feature-request.md

---

> name: "\U0001F4A1 Suggest an idea for this project"

Reviewer: ```suggestion
name: "\U0001F4A1 Suggest an idea or feature"
```

about: 'Before you start, check if an related

>   issue exists (https://github.com/redaxo/redaxo/issues) '

Reviewer: ```suggestion
  issue exists. See https://github.com/redaxo/redaxo/issues'
```

---

> **Feature description. / Feature Beschreibung **

Reviewer: Ich würde nur das hier lassen, rest kann weg finde ich

---

> name: "\U0001F4A1Feature request"

Reviewer: ```suggestion
name: "\U0001F4A1 Suggest an idea for this project"
```

> about: 'Suggest an idea for this project. Please, before you start, check if an related

Reviewer: ```suggestion
about: 'Before you start, check if an related
```

---
name: "\U0001F4A1 Suggest an idea or feature"

> about: 'Before you start, check if an related issue exists (https://github.com/redaxo/redaxo/issues) '

Reviewer: ```suggestion
about: 'Before you start, check if a related issue exists (https://github.com/redaxo/redaxo/issues) '
```

---

> **Feature description. / Feature Beschreibung **

Reviewer: ```suggestion
**Feature description / Feature Beschreibung **
```

Reviewer 2: Das letzte Leerzeichen müsste weg, ansonsten wird die Zeile nicht fett dargestellt.

---
name: "\U0001F4A1 Suggest an idea or feature"

> about: 'Before you start, check if a related issue exists. See 

Reviewer: ```suggestion
about: Before you start, check whether a similar suggestion was already made. Search via https://github.com/redaxo/redaxo/issues
```

> https://github.com/redaxo/redaxo/issues'

Reviewer: ```suggestion
```
